components/filter/test/filter.vdiff.js

```diff
@@ -191,6 +191,46 @@ describe('filter', () => {
 			});
 		});
 
+		it.only('select-all-then-clear', async() => {
```

Comment:
Are these `only` statements just temporary?

Reply:
Yup! Forgot to clean these up. Will fix